src/BlockEntities/BlockEntity.h

```diff
@@ -83,6 +84,10 @@ class cBlockEntity
 	Uses CopyFrom() to copy the properties. */
 	OwnedBlockEntity Clone(Vector3i a_Pos);
 
+	/** Returns the contents of this block entity that it would drop if broken.
+	Note that the block itself is not included; that's handled by the block handler. */
+	virtual cItems ConvertToPickups() const;
```

Comment:
This comment doesn't really reflect reality - it should probably be something like

 > Note that the block handler will usually handle the block itself, in addition to any blocks returned here.